administrator/components/com_users/src/Dispatcher/Dispatcher.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @package         Joomla.Administrator
+ * @subpackage      com_users
+ *
+ * @copyright   (C) 2017 Open Source Matters, Inc. <https://www.joomla.org>
```

Comment:
Should be 2021 right?


```suggestion
 * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
```